test.sh

```diff
@@ -37,7 +37,7 @@ function buildSample() {
         echo "+-------------------------------+"
         echo "| Executing tests using Gradle  |"
         echo "+-------------------------------+"
-        ./gradlew -Dscala.version="$MATRIX_SCALA" check
+        ./gradlew -Dscala.version="$MATRIX_SCALA" -Dbackend.server="$backend_server" clean check
```

Comment:
Added `clean` task is useful to local running to not think about previous compilation